Data/Scripts/aschar.as

```diff
@@ -2482,11 +2482,6 @@ void Update(int num_frames) {
             invincible = GetConfigValueBool("player_invincible");
         }
     }
-	
-	if(GetInputPressed(this_mo.controller_id, "k"))
-	{
-		SetOnFire(true);
-	}
```

Comment:
This can also be fixed up by editing your previous commit and doing a force push.

Reply:
Only do force pushes in PRs or branches, btw, and not ones that people may be working on. It's generally frowned upon because it requires people to pull your changes before they do anything else in that same branch.

Reply:
hello - sorry about the late response, i had my hands really full with the map jam and uni stuff

thank you for your feedback, i'll get around to updating this and my other pr's accordingly over this weekend ideally. might not be able to do all of them but at the very least most of the smaller ones, like this one, i should have done by sunday's end

i also just wanted to take this opportunity to say that i really do mean thank you for the feedback. 
deadass when i started this pr jazz i didn't even know what a pointer is or how classes worked lol. only just learnt what virtual functions are like, yesterday. so yeah, genuine apologies about the bugs and other "incompleteness-es" but i really appreciate the time you've taken to point them out and explain them 

i'll be sure to put extra care in my future prs